src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
```

Comment:
Please give clear variable names (for example, person)

Reply:
also you can reduce the amout of code

`
const women = people.filter((person) => withChildren
    ? people.some((child) => child.mother === person.name)
    : person.sex === 'f'
  )
`